apps/react/src/features/utils/no-empty.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Create a placeholder for empty value.
+ * @param value Value.
+ * @param placeholder Placeholder.
+ */
+export function noEmpty(value: unknown, placeholder?: string): string {
```

Comment:
Let's narrow the type here, in case you have `undefined`, you maybe want to pass `null` to be more explicit
```suggestion
export function noEmpty(value: string | null, placeholder?: string): string {
```